internal/pool/pool.go

```diff
@@ -835,7 +838,7 @@ func (p *ConnPool) removeConn(cn *Conn) {
 		// this can be idle conn
 		for idx, ic := range p.idleConns {
 			if ic.GetID() == cid {
-				internal.Logger.Printf(context.Background(), "redis: connection pool: removing idle conn[%d]", cid)
+				p.logger().Errorf(context.Background(), "redis: connection pool: removing idle conn[%d]", cid)
```

Comment:
I'm unsure about the error level here.

Should we decrease to a Warn or an Info ?